project.php

```diff
@@ -1840,52 +1836,50 @@ function do_smooth_reading()
                 "<b>$sr_deadline_str</b>"
             );
 
-            echo "<li>";
-            echo $sr_sentence;
-            echo "</li>\n";
+            echo "<p class='thin-para'>$sr_sentence</p>";
 
             if ($current_user_can_manage_SR_for_this_project)
             {
+                echo "<ul class='list-head'>";
                 echo "<li>";
                 $label = _('Extend Smooth Reading deadline by %1$s day(s) (between %2$d and %3$d).');
                 sr_echo_time_form($label, 1, 42, 1, true);
                 echo "</li>";
-                echo "<li>";
+                echo "<li class='list-head'>";
                 echo "<a href='$code_url/tools/upload_text.php?project=$projectid&stage=smooth_avail'>";
                 echo _("Replace the currently available Smooth Reading file.");
                 echo "</a>";
                 echo "</li>";
+                echo "</ul>";
             }
 
             if (!$project->PPer_is_current_user)
             {
-                echo_download_zip( _("Download zipped text for Smooth Reading"), '_smooth_avail' );
-
+                echo "<ul class='list-head'>";
+                echo_smoothreading_options($project);
                 // We don't allow guests to upload the results of smooth-reading.
                 global $user_is_logged_in;
                 if ( $user_is_logged_in )
                 {
-                    echo "<li>";
+                    echo "<li class='list-head'>";
                     echo "<a href='$code_url/tools/upload_text.php?project=$projectid&stage=smooth_done'>";
-                    echo _("Upload a smooth-read text") ;
+                    echo _("Upload a smoothread report") ;
```

Comment:
This should be `Upload a smooth read report`. Trying to standardize on one form for display instead of three, and at some point, it went to two separate words, no hyphen.

Reply:
I've changed this and some other similar cases.
There is an inconsistency where we have said "Upload a Smooth Read report" but "dowload Smooth Read _files_ uplaoded by xyz. The 'files' word is in a shared subroutine so more difficult to change.